org.adempiere.base/src/org/compiere/model/MRequest.java

```diff
@@ -1006,11 +1031,25 @@ else if (AD_Table_ID == MAsset.Table_ID)
 	 * @param AD_Table_ID
 	 * @param Record_ID
 	 * @param C_BPartner_ID
+	 * @deprecated - use {@link #newRequest(GridTab, int, int, String, int)} instead
 	 */
 	public static void newRequest(GridTab tab, int AD_Table_ID, int Record_ID, int C_BPartner_ID) {
+		newRequest(tab, AD_Table_ID, Record_ID, null, C_BPartner_ID);
+	}
+
+	/**
+	 * Create new request
+	 * @param tab Grid Tab for request
+	 * @param AD_Table_ID
+	 * @param Record_ID Record ID
+	 * @param Record_UU Record UUID
+	 * @param C_BPartner_ID
+	 */
+	public static void newRequest(GridTab tab, int AD_Table_ID, int Record_ID, String Record_UU, int C_BPartner_ID) {

```

Comment:
This is minor but all this recent changes actually make many changed API harder to use for developer. It kind of make the dev have to think now - must I always fill up both Record_ID and Record_UU parameter ? What happens when I fill up only one of them ? And are there situations where I must fill up both ?